src/tools/perf/ucp_tests.cc

```diff
@@ -381,6 +382,7 @@ class ucp_perf_test_runner {
         recv_datatype = ucp_perf_test_get_datatype(m_perf.params.ucp.recv_datatype,
                                                    m_perf.ucp.recv_iov, &recv_length,
                                                    &recv_buffer);
+        force_connection(ep);
```

Comment:
seems like this part is going to be included in test time. maybe just do it right after endpoint creation (even if test is not 1-sided)